app/Http/Requests/PasswordChangeRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PasswordChangeRequest extends FormRequest {
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules() {
+        return [
+            'password' => 'required|string|min:8',
+            'password_confirmation' => 'required|string|same:password',
+            'new_password' => 'required|string|min:8',
```

Comment:
We should require the user to type in new password twice but current password once

Reply:
Completed